acceptance_tests/ogcapi-features-schema.yaml

```diff
@@ -22,6 +22,8 @@ info:
   x-logo:
     url: 'https://www.ogc.org/pub/www/files/OGC_Logo_2D_Blue_x_0_0.png'
 servers:
+  - url: http://localhost/ogcapi
```

Comment:
just curious why we need this URL?

Reply:
For the tests :-)
We should have a better solution in the next versions...